ngraph/python/tests/test_onnx/test_additional_models.py

```diff
@@ -0,0 +1,50 @@
+# ******************************************************************************
+# Copyright 2018-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+import tests
+from operator import itemgetter
+from pathlib import Path
+import os
+
+from tests.test_onnx.utils import OpenVinoOnnxBackend
+from tests.test_onnx.utils.model_importer import ModelImportRunner
+
+MODELS_ROOT_DIR = tests.ADDITIONAL_MODELS_DIR
```

Comment:
```suggestion
onnx_home = os.path.expanduser(os.getenv('ONNX_HOME', os.path.join('~', '.onnx')))
MODELS_ROOT_DIR = os.path.join(onnx_home, 'additonal_models')
```

Reply:
I'd rather set the default value for `tests.ADDITIONAL_MODELS_DIR` to above value.